4_advanced_usage/40_node_properties.txt

```diff
@@ -0,0 +1,65 @@
+=== Node properties
+
+==== Defining properties
+
+Node properties can be found in the "properties" tab of each node in Rudder.
+
+Node properties can be modified using Rudder API, see http://www.rudder-project.org/rudder-api-doc/#api-Nodes-updateNodeProperties
```

Comment:
I cannot generate the doc at the moment, but it seems to work for other links in the documentation.